nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/http/InvokeHTTP.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.nifi.processors.standard;
+package org.apache.nifi.processors.standard.http;
```

Comment:
To avoid breaking changes when upgrading, the `InvokeHTTP` processor must remain in the same package.

Reply:
I've reverted it!